extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/binder/vertx/VertxTcpMetrics.java

```diff
@@ -35,7 +35,10 @@ public class VertxTcpMetrics extends VertxNetworkMetrics
     public Map<String, Object> connected(SocketAddress remoteAddress, String remoteName) {
         Map<String, Object> socketMetric = new HashMap<>();
         socketMetric.put(CONNECTED_SOCKET_SAMPLE,
-                LongTaskTimer.builder(nameConnections).register(registry).start());
+                LongTaskTimer.builder(nameConnections)
+                    .publishPercentileHistogram()
+                    .publishPercentiles(100)
+                    .register(registry).start());
         return socketMetric;
```

Comment:
I do not want to set this here.. as histograms are expensive. I know that means that exemplars will not be used OOTB, but we should not be incurring the data cost for all of these measurements. Folks should use meter filters to enable histograms for the metrics they want histograms for: this is a (runtime) user choice.